utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp

```diff
@@ -292,18 +372,35 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
 
     int resize_w = width, resize_h = height; // height and width
 
-    RocalImage image0 = rocalResize(handle, input1, resize_w, resize_h, false);
+    // RocalImage image0 = rocalResize(handle, input1, resize_w, resize_h, false);
+    RocalImage image0 = input1;
 
     RocalImage image1;
-
+    std::string interpolationTypeName,scaling_node_Name;
```

Comment:
Rename the variables properly
All should be in snake case.
scaling_node_Name - Do not use both small and capital letters